lib/index.js

```diff
@@ -310,23 +311,30 @@ Queue.prototype._reclaim = function(id) {
     queue: other.get(this.keys.QUEUE) || []
   };
 
+  var trackMessageIds = [];
+
+  var addConcatQueue = function(queue, incrementAttemptNumberBy) {
+    each(function(el) {
+      var id = el.id || uuid();
```

Comment:
this can go inside the `else` statement?

Reply:
Wanting to use the id inside of the if condition underneath. I could check if `el.id` is defined before this, but it feels better for consistency just to set it above and use `id` everywhere here